test/transpile-specs.js

```diff
@@ -17,26 +15,27 @@ const MOCHA = './node_modules/.bin/mocha';
 
 const readFile = B.promisify(fs.readFile);
 
-// we don't care about exec errors
-const exec = function exec (...args) {
-  return new B(function (resolve) {
+// we do not care about exec errors
+async function exec (...args) {
+  return await new B(function (resolve) {
+    // eslint-disable-next-line promise/prefer-await-to-callbacks
     cp.exec(args.join(' '), function (err, stdout, stderr) {
```

Comment:
what's the reason to still use callback here?

Reply:
Version promisified by Bluebird loses access to `stderr`, which is a problem. We need to manually promisify, which means callbacks.